ios/authentication.md

```diff
@@ -439,6 +439,16 @@ AWSMobileClient.sharedInstance().confirmSignIn(challengeResponse: "NEW_PASSWORD_
 AWSMobileClient.sharedInstance().signOut()
 ```
 
+### Global SignOut
+
+Using global signout, you can signout a user from all active login sessions.
```

Comment:
Please add the following:

By doing this, you are revoking all the OIDC tokens(id token, access token and refresh token) which means the user is signed out from all the devices. However, although the tokens are revoked, the AWS credentials will remain valid until they expire (which by default is 1 hour)